internal/extproc/translator/openai_gcpanthropic_stream.go

```diff
@@ -247,21 +270,22 @@ func (p *anthropicStreamParser) handleAnthropicStreamEvent(eventType []byte, dat
 				return nil, fmt.Errorf("received input_json_delta for unknown tool at index %d", event.Index)
 			}
 			index := int(event.Index)
+			args := ""
+			if len(event.Delta.PartialJSON) > 0 {
+				args = event.Delta.PartialJSON
+			}
```

Comment:
I think we can omit the if ?
```suggestion
			args := event.Delta.PartialJSON
```

Reply:
was testing something - removed it